ztests/suite/zqd/recruiter-curl.yaml

```diff
@@ -1,32 +1,29 @@
 script: |
+  source common.sh
   source recruiter-curl.sh
-  curl -X POST -d '{"addr":"a.b.c:5000","node_name":"a.b"}' \
-    http://$ZQD_HOST/recruiter/register 2> err
+  curl -X POST -d '{"addr":"a.b.c:5000","node_name":"a.b","timeout":1}' \
+     http://$ZQD_HOST/recruiter/register 2> err
+  curl -X POST -d '{"addr":"a.b.c:5000","node_name":"a.b","timeout":30000}' \
+    http://$ZQD_HOST/recruiter/register 2> err &
+  # This is intentionally backgrounded. The 100ms sleep allows the registration.
+  sleep 0.1
```

Comment:
Since our CI machines tend to be slow, can bump this up to at least a second? (It runs fine for me locally)

Reply:
Yes, I think I saw recruiter-curl fail once on GitHub. Must need a little more time there.